website/blog/2023-06-12-package-exports-support.md

```diff
@@ -0,0 +1,231 @@
+- React Native for Web projects using Metro will now be able to use the `"browser"` conditional export, removing the need for workarounds.
+
+Enabling Package Exports brings a few [edge-case breaking changes](#breaking-changes) that may affect specific projects, and which you can [test today](#validating-changes-in-your-project).
+
+**In a future React Native release, Package Exports will be enabled by default**. In a chicken-and-egg situation, React Native apps were previously a holdout for some packages to migrate to `"exports"` — or used our `"react-native"` root field escape hatch. Supporting these features in Metro will allow the ecosystem to move forward.
+
+### Enabling Package Exports (experimental)
+
+Package Exports can be enabled in your app's [**metro.config.js**](https://github.com/facebook/react-native/blob/0.72-stable/packages/react-native/template/metro.config.js) file via the [`resolver.unstable_enablePackageExports`](https://facebook.github.io/metro/docs/configuration/#unstable_enablepackageexports-experimental) option.
+
+```js
+const config = {
+  // ...
+  resolver: {
+    unstable_enablePackageExports: true,
+  },
+};
+```
+
+Metro exposes two further resolver options which configure how conditional exports behave:
+
+- [`unstable_conditionNames`](https://facebook.github.io/metro/docs/configuration/#unstable_conditionnames-experimental) — The set of [condition names](https://nodejs.org/docs/latest-v19.x/api/packages.html#community-conditions-definitions) to assert when resolving conditional exports. By default, we match `['require', 'import', 'react-native']`.
+- [`unstable_conditionsByPlatform`](https://facebook.github.io/metro/docs/configuration/#unstable_conditionsbyplatform-experimental) — The additional condition names to assert when resolving for a given platform target. By default, this matches `'browser'` when the platform is `'web'`.
+
+:::tip
+**Remember to use the React Native [Jest preset](https://github.com/facebook/react-native/blob/main/template/jest.config.js#L2)!** Jest includes support for Package Exports by default. In tests, you can override which `customExportConditions` are resolved using the [`testEnvironmentOptions`](https://jestjs.io/docs/configuration#testenvironmentoptions-object) option.
+:::
+
+#### Validating changes in your project
+
+For existing projects, we recommend that early adopters follow these steps to see if resolution changes occur after enabling `unstable_enablePackageExports`. This is a one-time process. It's likely that there will be no changes at all, but we'd like developers to opt in with certainty.
+
+<details>
+<summary>💡 Validating changes in your project</summary>
+
+1. Get all resolved dependencies (before changes):
+
+   ```sh
+   # Replace index.js with your entry file if needed, such as App.js
+   yarn metro get-dependencies index.js --platform android --output before.txt
+   ```
```

Comment:
We should also provide the `npx` (or `npm *` equivalent) commands since not everyone is using `yarn` and we're suggesting creating the new RN projects using `npx` in the first place.
So, something like:

```sh
# using npx
npx metro get-dependencies index.js --platform android --output before.txt

# OR using yarn
yarn metro get-dependencies index.js --platform android --output before.txt
```

Or a note at the start about how to switch to `npx` commands would be great.

P.S.: I know some previous blog posts have been using `yarn` only, which also somewhat conflicts with rest of the decisions made for the docs.

Cc: @NickGerleman